### PR TITLE
chore(release): Prepare v0.25.1

### PR DESCRIPTION
**Pull request by jszwedko**
TODOs:

- [x] Pick https://github.com/vectordotdev/vector/pull/15112

### CODE REVIEW

File: website/cue/reference/releases/0.25.0.cue [hunk 1]
```diff
@@ -9,11 +9,11 @@ releases: "0.25.0": {
...
 			Vector fails to load configurations using environment variables outside of string values
-			for configuration options. This will be fixed in 0.25.1.
+			for configuration options. Fixed in 0.25.1.
 			""",
 		"""
 			Vector fails to load multi-file configurations using the global `timezone` configuration
-			option. This will be fixed in 0.25.1.
+			option. Fixed in 0.25.1.
 			""",
 	]
```
Comment (neuronull) [inline website/cue/reference/releases/0.25.0.cue:18]: :thinking: does this need to merge in https://github.com/vectordotdev/vector/commit/a62e7d64954321c67bab546dc21b9e3abc4be069 ?

Comment (jszwedko) [reply website/cue/reference/releases/0.25.0.cue]: I was actually just doing this but forgot to mention it in the TODOs. Good spot.

**Comment by neuronull** [comment]
Looks good to me but for picking the TODO item + adding it to the `website/cue/reference/releases/0.25.1.cue`

File: website/cue/reference/releases/0.25.1.cue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+				Vector now loads multi-file configurations using the global `timezone` configuration
+				option again.
+				"""
+			pr_numbers: [15077]
+		},
+		{
+			type: "fix"
+			scopes: ["config"]
+			description: """
+				The `prometheus_remote_write` sink now allows `auth.bearer` to be provided again.
```
Comment (neuronull) [inline website/cue/reference/releases/0.25.1.cue:38]: :thinking:  should this wording be more clear that its a config option?
```suggestion
				The `prometheus_remote_write` sink now allows the configuration `auth.bearer` to be provided again.
```

File: website/cue/reference/releases/0.25.0.cue [hunk 3]
```diff
@@ -9,15 +9,15 @@ releases: "0.25.0": {
...
 			""",
 		"""
 			Vector fails to load multi-file configurations using the global `timezone` configuration
-			option. This will be fixed in 0.25.1.
+			option. Fixed in 0.25.1.
 			""",
 		"""
 			The `prometheus_remote_write` sink doesn't support specifying `auth.bearer` as it
-			should. This will be fixed in 0.25.1.
+			should. Fixed in 0.25.1.
```
Comment (neuronull) [inline website/cue/reference/releases/0.25.0.cue:20]: Perhaps the same on the line above this though I can't suggest since it wasn't edited here.

**Review by neuronull — APPROVED** [review]

**Pull request merged**